docs/design/mp_arktos.md

```diff
@@ -0,0 +1,145 @@
+<!--
+SPDX-License-Identifier: MIT
+Copyright (c) 2020 The Authors.
+
+Authors: Sherif Abdelwahab <@zasherif>
+         Phu Tran          <@phudtran>
+         Catherine Lu      <@clu2>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:The above copyright
+notice and this permission notice shall be included in all copies or
+substantial portions of the Software.THE SOFTWARE IS PROVIDED "AS IS",
+WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+# Arktos Network and Mizar Integration
+
+## Introduction
+
+This document is designated for Arktos service using Mizar as its underline network system. In this document, you will find the design detail on how to integration Arktos multi-tenancy network object with Mizar as its network platform, so that Arktos can fully utilize network objects offered in Mizar, including vpcs, nets, services, pods, bouncers and dividers. 
+
+## Arktos Network Object Overview: 
+The Arktos network object contains its type and type-specific configurations, which translates into spec fields type and vpcID:
+```yaml
+spec:
+  type: mizar
+  vpcID: vpc-1a2b3c4d
+```
+The type field is the only required field, where it identifies the underline network service (which in our case is mizar). VpcID field is optional, and it lists the vpc id that is associated with this Arktos network object. If vpcID is not specified, the default VPC will be used. 
+
+The Arktos network object and VPC have **one to one relationship**, where one Arktos network object only targets to one VPC in Mizar. In other words, Arktos network object is equivalent to a vpc object in Mizar. 
+
+## Mizar VPC Object Overview: 
+In Mizar, VPC object has two required fields: ip and prefix, where in combination, they compose the CIDR of the VPC. For example, for CIDR 10.0.0.0/16, the ip = 10.0.0.0 and prefix = 16. 
+```yaml
+spec:
+  ip: 10.0.0.0
+  prefix: 16
+```
+## Proposal 
+Given the fact that Arktos network object is equivalent to vpc object in Mizar, I would like to propose two options: 
```

Comment:
In both option1 and option2, can you give more details how are we going to create a mizar network within the VPC? I remember you mentioned passing multiple CIDR blocks.

Reply:
That's when we ask Arktos team to provide VPC CIDR, and they said they can provide subnet CIDR in the annotation section like this: 

type: Network
metadata:
  name: network-foo
  annotations:
     subnets: ["192.168.0.0/24", "192.168.1.0/24"]
spec:
  type: mizar  // vpc is not a key; put mizar or the value that network provider recognizes
  vpcID: mizar-12345

But above format will not work since we still need vpc CIDR range. Two subnet CIDR's do not tell us the CIDR of the VPC.